manifests/supervisorcluster/1.29/cns-csi.yaml

```diff
@@ -95,7 +95,7 @@ rules:
     verbs: ["get", "list", "watch"]
   - apiGroups: [ "snapshot.storage.k8s.io" ]
     resources: [ "volumesnapshots" ]
-    verbs: [ "get", "list", "patch" ]
+    verbs: [ "get", "list", "patch", "update", "watch" ]
```

Comment:
why do we need  a watch on volumesnapshots?

Reply:
we needed it to be handled by reconciler, without this permission, reconciler did not start correctly